### DISCUSSION TITLE
求高手帮忙

### DISCUSSION THREAD

**Discussion by RiseToTop** [ant-design/ant-design#46168] [Q&A]
{
name:"xxx",
path:"xxx",
component: './tailor'
}
这是我动态获取的菜单数据，菜单正常显示，但一点击就404,是哪里写的不对吗，用的是antd pro和umijs4

**Comment by yoyo837** [comment]
v抱歉， 这种提问没有任何人能帮忙，太抽象。还得来回问很多问题，不如丢个zip来得直接。

**Comment by promiseLC** [comment]
我猜的你是你动态路由根本没有配置成功,你获取到数据,自己再看看路由相关的地方有没有重新执行

**Comment by komorebi-cqd** [comment]
获取了动态菜单，你本地也要有这个菜单的路由的哦
